create-vite-express/templates/react-ts/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/react-dom": "^18.2.16",
     "@vitejs/plugin-react": "^4.2.0",
     "nodemon": "^3.0.1",
+    "tsx": "^4.3.0",
```

Comment:
`tsx` should be a dependency like `ts-node` was because it is used in production mode.